src/JwtGuardianVerifier.sol

```diff
@@ -226,8 +226,6 @@ contract JwtGuardianVerifier is IGuardianVerifier, Initializable {
         // Parse the extra data
         JwtData memory jwtData = abi.decode(proof.data, (JwtData));
 
-        require(jwtData.isCodeExist == true, CodeDoesNotExist());
```

Comment:
@zkfriendly with the jwt verifier, what should we do with this check? For example, in the email recovery EmailAuth flow, we assert `emailData.isCodeExist == true` on the acceptance part of the flow, but not in the handle recovery part of the flow

Also related to the question I raised on your fixture PR

Reply:
As far as I understand, this was a check for account code in the command, which doesn't exist here. The args are still passed as the jwt-verifier expects similars args as email-verifier.

Reply:
Will see what zkfriendly says, if we don't have a concrete answer on whether the check should be here or not, we can merge the pr

Reply:
We've discussed this in [another PR](https://github.com/zkemail/email-tx-builder/pull/127#discussion_r2045109640) and decided we need to think more about this check. This PR is good to go